frontend/cypress/integration/login/login.spec.js

```diff
@@ -0,0 +1,47 @@
+/// <reference types="cypress" />
+
+context('Login test', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:4000/login');
+  });
+
+  it('should show register button', () => {
+    cy.get('#btn-register');
+  });
+
+  it('should show and type email', () => {
+    cy.get('#email')
+      .type('admin@gmail.com');
+  });
+
+  it('should show and type password', () => {
+    cy.get('#password')
+      .type('123456');
+  });
+
+  it('try login with invalid user', () => {
+    cy.get('#email')
+      .type('invaliduser@gmail.com');
+
+    cy.get('#password')
+      .type('maybeiscorrectpassword');
+
+    cy.get('#btn-login')
+      .click();
+
+    cy.url().should('not.be.include', '/main');
+  });
```

Comment:
You could also intercept the request to check if it returns an error
https://docs.cypress.io/api/commands/intercept#Intercepting-a-response